test/browser.test.js

```diff
@@ -9,13 +9,21 @@ var vm = require('vm');
 var createBrowserLikeContext = require('./helpers/browser').createContext;
 var printContextLogs = require('./helpers/browser').printContextLogs;
 
+function addPlugins(b) {
+  var files = fs.readdirSync(path.join(__dirname, '../lib/plugins'));
+  files.forEach(function(f) {
+    b.require('../../../lib/plugins/' + f,
+      { expose: './plugins/' + path.basename(f, '.js') });
+  });
+}
```

Comment:
Ha, looks like my anticipation in https://github.com/strongloop/loopback-boot/pull/181/files#r63145820 was right. Can we move this to `boot.compileToBrowserify` please?